DX-390 (#123)
SwedbankPay/developer.swedbankpay.com: payments/vipps/seamless-view.md

  authorization.
* ⑥ The response will contain the state of the transaction. It will normally be

>   in `AwaitingActivity` in this fase of the payment.

Reviewer: ```suggestion
  `AwaitingActivity` in this phase of the payment, meaning we are awaiting a response from Swish.
```

Author: Why are we waiting for response from Swish in the documentation for Vipps?

Reviewer: Good question. I didn't notice both Swish and Vipps were tackled in the same PR. Replace `Swish` with `Vipps` where appropriate.

Author: It was not, so I don't know why you suddenly took `Vipps`. But it's handled now.

Reviewer: Seems like the following line 92 is still in the file even though the new description is added on lines 93 and 94?

> It will normally be in `AwaitingActivity` in this fase of the payment.

>   `AwaitingActivity` means that we are waiting for response from external

Reviewer: ```suggestion
```

>   systems.

Reviewer: ```suggestion
```
